### PR TITLE
Rc update

### PR DESCRIPTION
**Pull request by klutvott123**
Updates feedforward and rc smoothing settings for BF 4.3 according to https://github.com/betaflight/betaflight/pull/10723 and https://github.com/betaflight/betaflight/pull/10650

screenshot_x7_21-05-24_22-57-29
screenshot_x7_21-05-24_22-56-49
screenshot_x7_21-05-24_22-56-43

### CODE REVIEW

File: src/SCRIPTS/BF/PAGES/rx.lua [hunk 1]
```diff
@@ -23,29 +23,39 @@ if apiVersion >= 1.016 then
...
 
-if apiVersion >= 1.040 then
-    labels[#labels + 1] = { t = "Input Filter",      x = x,          y = inc.y(lineSpacing) }
-    fields[#fields + 1] = { t = "Cutoff",            x = x + indent, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 255, vals = { 26 }, table = { [0] = "Auto" } }
-    fields[#fields + 1] = { t = "Type",              x = x + indent, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 1, vals = { 28 }, table = { [0] = "PT1", "BIQUAD"} }
-    labels[#labels + 1] = { t = "Derivative Filter", x = x,          y = inc.y(lineSpacing) }
-    fields[#fields + 1] = { t = "Cutoff",            x = x + indent, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 255, vals = { 27 }, table = { [0] = "Auto" } }
-    fields[#fields + 1] = { t = "Type",              x = x + indent, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 3, vals = { 29 }, table = { [0] = "Off", "PT1", "BIQUAD", "Auto"} }
-end
+    if apiVersion >= 1.040 then
```
Comment (haslinghuis) [inline src/SCRIPTS/BF/PAGES/rx.lua:47]: Can this block be moved to line 39?

Comment (klutvott123) [reply src/SCRIPTS/BF/PAGES/rx.lua:47]: Sure. Usually we're more worried about getting things in the "right" order and that sometimes means multiple checks for the same api version. In this case it's fine if we combine them. The order of things won't make any less sense than it currently does. lol

File: src/SCRIPTS/BF/PAGES/pids2.lua [hunk 2]
```diff
@@ -48,15 +48,13 @@ if apiVersion >= 1.040 then
...
-end
-
-if apiVersion >= 1.044 then
-    labels[#labels + 1] = { t = "FF Interpolate", x = x,          y = inc.y(lineSpacing) }
-    fields[#fields + 1] = { t = "Mode",           x = x + indent, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 4, vals = { 51 }, table = { [0] = "OFF", "ON", "AVG_2", "AVG_3", "AVG_4" } }
-    fields[#fields + 1] = { t = "Smoothness",     x = x + indent, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 75, vals = { 52 } }
-    fields[#fields + 1] = { t = "Boost",          x = x + indent, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 50, vals = { 53 } }
+    labels[#labels + 1] = { t = "Feedforward",    x = x,          y = inc.y(lineSpacing) }
+    fields[#fields + 1] = { t = "Transition",     x = x + indent, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 100, vals = { 9 }, scale = 100 }
+    if apiVersion >= 1.044 then
```
Comment (haslinghuis) [inline src/SCRIPTS/BF/PAGES/pids2.lua:53]: lol, the conditional is not needed anymore.

Comment (klutvott123) [reply src/SCRIPTS/BF/PAGES/pids2.lua:53]: Yes it is. We want to show these parameters for api version 1.044 and up.

Comment (haslinghuis) [reply src/SCRIPTS/BF/PAGES/pids2.lua:53]: Oops color blindness, reading too long I guess :man_facepalming:

**Review by haslinghuis — APPROVED** [review]

File: src/SCRIPTS/BF/PAGES/pids2.lua (hunk 2 above)
Comment (haslinghuis) [reply src/SCRIPTS/BF/PAGES/pids2.lua:53]: Not sure how I could misread the removed part as valid part of the code. :man_palm

**Review by mikeller — APPROVED** [review]

**Pull request merged**